api_tests/src/user.spec.ts

```diff
@@ -198,4 +199,14 @@ test("Set a new avatar, old avatar is deleted", async () => {
   // make sure only the new avatar is kept
   const listMediaRes3 = await alphaImage.listMedia();
   expect(listMediaRes3.images.length).toBe(1);
+
+  // Now try to save a user settings, with the icon missing,
```

Comment:
Added a test here to make sure that it isn't clearing the avatar, when that field is missing.